ironfish-cli/scripts/import-export-test/0p1p73_mnemonic.txt

```diff
@@ -0,0 +1 @@
+bicycle game eight minimum rival shoulder airport birth section flash walnut seat omit front carpet cheap wall arch brick wink kangaroo safe gas hub
```

Comment:
This is ok, but since its just a spend key we don't need to include it since the spend key format didn't change only the ones with metadata.

Reply:
I'll remove this one. I think we could probably remove the mnemonic test cases from some of the earlier versions to reduce the test duration